File: test-suite/golden-tests/multiple-instances-1-r/rms.R
# the smart way
rms1 <- function(xs){
  sqrt(mean(xs^2))
}

# the dumb way
rms2 <- function(xs){
  result = 0
  for (x in xs){
    result = result + x ^ 2
  }
  result = result / length(result)
  result = sqrt(result)
  return(result)
}

File: test-suite/golden-tests/multiple-instances-2-r/rms.py
import math

def rms1(xs):
    return math.sqrt(sum([x*x for x in xs])/len(xs))

def rms2(xs):
    y = 0
    for x in xs:
        y += x*x
    y = y / len(xs)
    return math.sqrt(y)
